ios/adapters/epson/NetPrinterEpsonAdapter.m

```diff
@@ -1,112 +1,160 @@
+
 //
 //  NetPrinterEpsonAdapter.m
 //
 //  Created by Till POS on 14/09/21.
 //
 
-
 #import "NetPrinterEpsonAdapter.h"
-#import "EpsonPrinterSDK.h"
-#import "../../utils/NSDataAdditions.h"
 #import "../../utils/EpsonUtils.h"
+#import "../../utils/NSDataAdditions.h"
+#import "EpsonPrinterSDK.h"
 #import <stdlib.h>
 
-@interface NetPrinterEpsonAdapter() <Epos2PtrReceiveDelegate>
-- (void) connectAndSendAux:(NSString *_Nullable)host printRawData: (NSString *_Nullable)text success:(RCTResponseSenderBlock _Nullable )successCallback fail:(RCTResponseSenderBlock _Nullable )errorCallback;
+@interface NetPrinterEpsonAdapter () <Epos2PtrReceiveDelegate>
+- (void)connectAndSendAux:(NSString *_Nullable)host
+             printRawData:(NSString *_Nullable)text
+                  success:(RCTResponseSenderBlock _Nullable)successCallback
+                     fail:(RCTResponseSenderBlock _Nullable)errorCallback
+               printerObj:(Epos2Printer *)printer;
```

Comment:
Assuming here `printer` object will be cleaned up after the function `connectAndSendAux` is finished